conans/test/integration/command/install/install_test.py

```diff
@@ -363,16 +363,16 @@ def test_install_disabled_remote(client):
     client.run("create . Pkg/0.1@lasote/testing")
     client.run("upload * --confirm --all -r default")
     client.run("remote disable default")
-    client.run("install Pkg/0.1@lasote/testing -r default", assert_error=True)
-    assert "ERROR: Remote 'default' is disabled" in client.out
+    client.run("install Pkg/0.1@lasote/testing -r default")
+    assert "Pkg/0.1@lasote/testing: Already installed!" in client.out
```

Comment:
Note this behavior change, could be considered a bug, why would fail if not really hitting the server at all?